src/java/picard/analysis/CollectGcBiasMetrics.java

```diff
@@ -82,13 +71,6 @@
     @Option(shortName = StandardOptionDefinitions.REFERENCE_SHORT_NAME, doc = "The reference sequence fasta file.")
```

Comment:
You're right, I'm not sure why it was before. It gave me an error when I had others that were duplicates but didn't have a problem with that one. I'll remove it.